csharp/test/Ice/info/AllTests.cs

```diff
@@ -148,7 +148,7 @@ public static void Run(TestHelper helper)
                 testIntf = ITestIntfPrx.Parse(helper.GetTestProxy("test", 0), communicator);
             }
 
-            string defaultHost = communicator.GetProperty("Ice.Default.Host") ?? "";
+            string defaultHost = communicator.GetProperty("Test.Host")!;
```

Comment:
I think it would make more sense to use:
```
string host = helper.GetTestHost();
```

and keep the Test.Host property inside TestHelper... not "leak" it to all the tests.